### ISSUE TITLE
It jackpine i18n ready?

### ISSUE THREAD

**Issue by ibes** [45-North-Ventures-LLC/jackpine#14]
Will strings like here be translated in other places?
https://github.com/45-North-Ventures-LLC/jackpine/blob/master/archive.php#L16

There is no __() present, so I am wondering if that was missed or if there is magic somewhere else.

**Comment by szepeviktor** [comment]
Yes, in the `next` branch: https://github.com/45-North-Ventures-LLC/jackpine/blob/next/theme/archive.php#L15-L17
